### ISSUE TITLE
Vehicle Noclip if sudden change in render distance from low to high

### ISSUE THREAD

**Issue by xdeadstokex** [Unknown025/Flans-Mod-Plus#217]
Maybe  due to the machine cant not catch up with render and physic at the same time bcs this only happen at intense lag or no performance mods at all (the case below is less servere than what I encounter first time, basically the heli noclip and fly around like crazy)

https://github.com/user-attachments/assets/5d501411-c81c-4d7f-a35b-49c819190b29

**Comment by GoldSloth** [comment]
Yeah, this is entirely client side. It's increased the entity tracking distance to start rendering and ticking the vehicles but the ground hasn't rendered in yet. I don't think there's an easy solution for this, and it's fairly minor tbh.

**Comment by xdeadstokex** [comment]
funny evidence
![image](https://github.com/user-attachments/assets/f09643fe-5eea-4b7d-a921-95123e72ccbb)
